src/content/articles/en/how-to-slack.mdx

```diff
@@ -0,0 +1,187 @@
+---
+id: 202003101
+title: "How-To Slack"
+date: 2020-03-10T18:42:25+00:00
+author: Rathes Sachchithananthan
+template: post
+image: ../../images/blog/slack-rgb-white.png
+description: Here, in this article, I am going to share with you a few tips and tricks that I have collected over the years that will make distributed teams working better together using the power of Slack.
+categories:
+  - Web
+tags:
+  - Slack
+locale: en_US
+---
+
+import Note from "../../../components/Note"
+
+When working in distributed companies and teams it is even more important that you communicate more often and more efficiently. And it is not always an easy task but there are plenty of tools to support you. One of them is Slack who have changed the way we work a lot — even in co-located teams.
+
+But with every feature added to Slack, it also adds up the risk that Slack becomes the new email in your company: Another application with an unorganized pile of messages and information that you can't handle anymore and that does not only not support you but even makes you less productive.
+
+Here, in this article, I am going to share with you a few tips and tricks that I have collected over the years that will make distributed teams working better together using the power of Slack.
+
+<Note>
+  While this article focusses on Slack, you can apply most of the ideas to other
+  communication channels as well such as Microsoft Teams.
+</Note>
+
+## Short-lived communication
+
+As mentioned above, it is really important that you communicate often. It is better that you post in Slack too much than too less.
+
+At the same time, keep in mind that Slack is a place for short-lived communication. While the search functionality of Slack is pretty good and you can find things back really easily, you still need to know what you have to search for. You as the author might know but what about the others? Or even newbies?
+
+So make sure that you separate the information that has a long-term value from your day-to-day communication. Make sure that everything that you might need later is organized in a knowledge base. Tools like [Notion](https://www.notion.so/) or [Slite](https://slite.com/) might help you with that (or even [Confluence](https://www.atlassian.com/software/confluence) if you like to do it the painful way).
+
+Everything else can be communicated via Slack.
+
+## Your own Slack etiquette
+
+Slack is easy to use since it is just another messenger just like for example the Facebook Messenger. But can you therefore behave the same way? It is important that your work out a few rules and guidelines in your company. These guidelines will make sure that, while you have that team spirit and camaraderie, things do not get out of hands.
```

Comment:
```suggestion
Slack is easy to use since it is just another messenger just like for example the Facebook Messenger. But can you, therefore, behave the same way? It is important that your work out a few rules and guidelines in your company. These guidelines will make sure that, while you have that team spirit and camaraderie, things do not get out of hands.
```